src/js/game/LevelMVC/LevelModel.js

```diff
@@ -597,9 +597,29 @@ module.exports = class LevelModel {
         return false;
       }
     }
+
+    if (this.checkConflict(this.getMoveDirectionPosition(entity, direction), plane)) {
+      return false;
+    }
     return this.getPlaneToPlaceOn(this.getMoveDirectionPosition(entity, direction), entity) !== null;
   }
 
+  checkConflict(position, placementPlane) {
+    var conflict = false;
+    this.controller.levelEntity.entityMap.forEach ( workingEntity => {
```

Comment:
Ah sorry, this should be `forEach(workingEntity` — no space before or after the paren.